packages/forma-36-react-components/src/components/Asset/AssetIcon/README.mdx

```diff
@@ -0,0 +1,79 @@
+import { AssetIcon } from './AssetIcon';
+
+# AssetIcon
+
+The AssetIcon component is a visual representation of an asset type such as an archive or a video.
+
+<AssetIcon type="archive" />
+
+## Table of contents
+
+- [Variations](#variations)
+  - [Archive](#archive)
+  - [Audio](#audio)
+  - [Code](#code)
+  - [Image](#image)
+  - [Markup](#markup)
+  - [PDF](#pdf)
+  - [Plaintext](#plaintext)
+  - [Presentation](#presentation)
+  - [Rich text](#rich-text)
+  - [Spreadsheet](#spreadsheet)
+  - [Video](#video)
+- [Usage](#usage)
+
+## Variations
+
+AssetIcon can represent the following asset types:
+
+### Archive
+
+<AssetIcon type="archive" />
+
+### Audio
+
+<AssetIcon type="audio" />
+
+### Code
+
+<AssetIcon type="code" />
+
+### Image
+
+<AssetIcon type="image" />
+
+### Markup
+
+<AssetIcon type="markup" />
+
+### PDF
+
+<AssetIcon type="pdf" />
+
+### Plaintext
+
+<AssetIcon type="plaintext" />
+
+### Presentation
+
+<AssetIcon type="presentation" />
+
+### Rich text
+
+<AssetIcon type="richtext" />
+
+### Spreadsheet
+
+<AssetIcon type="spreadsheet" />
+
+### Video
+
+<AssetIcon type="video" />
+
+## Usage
```

Comment:
We could call this `Example of usage`, just to keep it consistent with other README's. WDYT?